Content.Shared/Chemistry/Solutions/Solution.cs

```diff
@@ -0,0 +1,880 @@
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FixedPoint;
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+using System.Collections;
+using System.Linq;
+
+namespace Content.Shared.Chemistry.Solutions;
+
+/// <summary>
+///     A solution of reagents.
+/// </summary>
+[Serializable, NetSerializable]
+[DataDefinition]
+public sealed partial class Solution : IEnumerable<ReagentQuantity>, ISerializationHooks
+{
```

Comment:
All of the namespaces changes should probably be a separate PR because it makes reviewing much harder.

Reply:
Moved the namespace changes back.